tests/e2e/test_sampling_params.py

```diff
@@ -0,0 +1,272 @@
+import pytest
+from spyre_util import get_spyre_backend_list, get_spyre_model_list
+from vllm import LLM, SamplingParams
+
+
+@pytest.fixture
+@pytest.mark.parametrize("model", get_spyre_model_list())
+def spyre_model(model: str, max_model_len: int, max_num_seqs: int,
+                block_size: int, tensor_parallel_size: int) -> LLM:
+    return LLM(
+        model=model,
+        tokenizer=model,
+        max_model_len=max_model_len,
+        max_num_seqs=max_num_seqs,
+        block_size=block_size,
```

Comment:
You can remove the `tokenizer`, `block_size`, `max_num_seqs` and `tensor_parallel_size`. The defaults are fine.